tests/test_custom_models.py

```diff
@@ -2820,6 +2820,51 @@ def test_requires_grad_lora_different_targets(self):
             "base_model.model.lin1.lora_B.adapter1.weight",
         )
 
+    def test_requires_grad_qalora_same_targets(self):
```

Comment:
Hmm, for some reason I haven't really noticed this test. IMO it is not necessary and can be removed. This is because QALoRA does not change the handling of multiple adapters, so there is not really a reason to believe there could be anything wrong there. We also don't check other quantization methods here. Unless you had a specific reason to add this test, I'd suggest to just remove it.